### ISSUE TITLE
The UTC time represented when the offset is applied must be between year 0 and 10,000. Parameter name: offset

### ISSUE THREAD

**Issue by ghuntley** [NuGet/Home#3089]
# Problem

Package restore fails w/Mono and NuGet 3.4.3.0 (i.e. current version in Xamarin Studio 6.1 alpha channel). I've also observed this with NuGet v3.x during the restoring addins step with Cakebuild.

```
Installing Xamarin.Android.Support.v7.CardView 23.3.0.
Error downloading 'Xamarin.Android.Support.v7.CardView.23.3.0' from 'https://api.nuget.org/v3-flatcontainer/xamarin.android.support.v7.cardview/23.3.0/xamarin.android.support.v7.cardview.23.3.0.nupkg'.
The UTC time represented when the offset is applied must be between year 0 and 10,000.
Parameter name: offset
Unable to find version '23.3.0' of package 'Xamarin.Android.Support.v7.CardView'.
  https://api.nuget.org/v3/index.json: Error downloading 'Xamarin.Android.Support.v7.CardView.23.3.0' from 'https://api.nuget.org/v3-flatcontainer/xamarin.android.support.v7.cardview/23.3.0/xamarin.android.support.v7.cardview.23.3.0.nupkg'.
  The UTC time represented when the offset is applied must be between year 0 and 10,000.
  Parameter name: offset
```
# Similar issues
- https://github.com/NuGet/Home/issues/1984
- https://github.com/NuGet/Home/issues/2020
# Environment

```
=== Xamarin Studio Business ===

Version 6.1 (build 4963)
Installation UUID: b5ee7f0c-ce10-4f92-94cd-449c031e0047
Runtime:
    Mono 4.4.0 (mono-4.4.0-branch/fcf7a6d) (64-bit)
    GTK+ 2.24.23 (Raleigh theme)

    Package version: 404000148

=== NuGet ===

Version: 3.4.3.0

=== Xamarin.Profiler ===

Not Installed

=== Xamarin.Android ===

Version: 6.1.99.224 (Xamarin Business)
Android SDK: /Users/ghuntley/Library/Developer/Xamarin/android-sdk-macosx
    Supported Android versions:
        6.0 (API level 23)

SDK Tools Version: 25.1.7
SDK Platform Tools Version: 24.0.0
SDK Build Tools Version: 23.0.3

Java SDK: /usr
java version "1.7.0_71"
Java(TM) SE Runtime Environment (build 1.7.0_71-b14)
Java HotSpot(TM) 64-Bit Server VM (build 24.71-b01, mixed mode)

Android Designer EPL code available here:
https://github.com/xamarin/AndroidDesigner.EPL

=== Xamarin Android Player ===

Version: 0.6.5
Location: /Applications/Xamarin Android Player.app

=== Apple Developer Tools ===

Xcode 7.3.1 (10188.1)
Build 7D1014

=== Xamarin.Mac ===

Version: 2.9.2.81 (Xamarin Business)

=== Xamarin.iOS ===

Version: 9.9.0.288 (Xamarin Business)
Hash: ef65ca5
Branch: master
Build date: 2016-06-07 21:27:36-0400

=== Xamarin Inspector ===

Version: 0.9.0.14
Hash: 4d868da
Branch: master
Build date: Mon Jun 13 19:14:13 UTC 2016

=== Build Information ===

Release ID: 601004963
Git revision: b37c52e0249c009e308eb144957ac97d1003ec0e
Build date: 2016-06-10 12:00:02-04
Xamarin addins: db4bae9b11a32d16ee009695cd1bfa36455187b0
Build lane: monodevelop-lion-cycle8-preview2

=== Operating System ===

Mac OS X 10.11.5
Darwin geoffrey-mac 15.5.0 Darwin Kernel Version 15.5.0
    Tue Apr 19 18:36:36 PDT 2016
    root:xnu-3248.50.21~8/RELEASE_X86_64 x86_64

=== Enabled user installed addins ===

xUnit.NET 2 testing framework support 0.6.6
Redth's Addins 1.0.2
Gorilla Player 0.9.1.2
Xamarin Inspector 0.9.0.14

```

**Comment by mrward** [comment]
This is a [bug](https://bugzilla.xamarin.com/show_bug.cgi?id=37970) in Mono's zip implementation.

I added a fix to Xamarin Studio 6.1 to workaround this but unfortunately this fix is not in the latest release of Xamarin Studio 6.1 on the alpha channel. Using NuGet 3.4 from the command line requires the Mono bug fix.

A workaround is to switch your local time zone to be UTC+0 or a negative offset. For the NuGet command line a workaround is to switch to using NuGet v2.

**Comment by donnib** [comment]
@mrward any idea when can expect a release of mono version that will solve this ? As far as i can see this bug is in master in Mono.

**Comment by mrward** [comment]
The bug is fixed in the master branch of Mono. It should be in the next service release of Mono which is [currently scheduled for a stable release](https://releases.xamarin.com/beta-release-cycle-7-service-release-1/) in July.

**Comment by tahmidmozaffar** [comment]
Any update for this issue? Is it released?

**Comment by mrward** [comment]
@receme - Looking at the commits the fix should be in Mono 4.4.2.4 which is currently available from Xamarin's beta channel.

**Comment by PatoBeltran** [comment]
Closing as Mono version has already been released.
